recipes/simbody/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = "3.5.3" %}
+{% set variant = "openblas" %}
+
+package:
+    name: simbody
+    version: {{ version }}
+
+source:
+    fn: Simbody-{{ version }}.tar.gz
+    url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
+    sha256: 8005fbdb16c6475f98e13b8f1423b0e9951c193681c2b0d19ae5b711d7e24ec1
+
+build:
+    number: 200
+    skip: true                # [win]
+    features:
+        - vc14                # [win and py35]
+        - blas_{{ variant }}  # [unix]
+    # NOTE : This is needed to ensure the paths to Simbody's binaries, like
+    # simbody-visualizer, are corrected.
+    detect_binary_files_with_prefix: true
+
+# NOTE: This installs Simbody and the optional visualizer on Linux, Windows, and
+# OSX. We build against the openblas available in conda-forge which includes
+# lapack on Linux and OSX. blas and lapack are not needed on Windows because
+# Simbody includes them. For the visualizer, FreeGLUT is available via
+# anaconda.org and conda-forge but libXi and libXmu are not, so the user will
+# need those to be installed via their Linux package manager. Glut is included
+# with Simbody for Windows and OSX includes a Glut implementation. TODO:
+# Doxygen is the only dependency that is omitted here, so the rendered
+# documentation is not built.
+requirements:
+    build:
+        - toolchain
+        - cmake
+        - blas 1.1 {{ variant }}  # [not win]
+        - openblas 0.2.18*        # [not win]
+        - freeglut                # [linux]
+        - python                  # [win]
+    run:
+        - blas 1.1 {{ variant }}  # [not win]
+        - openblas 0.2.18*        # [not win]
+        - freeglut                # [linux]
+test:
+  requires:
+    - python 3.5.*  # [win and py35]
+  commands:
+    - if not exist %LIBRARY_BIN%\\simbody\\simbody-visualizer exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\Simbody.h exit 1           # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKcommon.h exit 1       # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKlapack.h exit 1       # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKmath.h exit 1         # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKsimbody_aux.h exit 1  # [win]
+    - if not exist %LIBRARY_INC%\\simbody\\SimTKsimbody.h exit 1      # [win]
```

Comment:
A user should *not* use `#include <simbody/Simbody.h>`, and in general should not be able to do so unless they install Simbody into `/usr`.  As shown in Simbody [examples](https://github.com/simbody/simbody/blob/master/examples/ChainExample.cpp#L32), the user should use `#include "Simbody.h"` (or `#include <Simbody.h>`). This will work cross-platform because of the way we create a cmake [Config file](https://github.com/simbody/simbody/blob/master/cmake/SimbodyConfig.cmake.in#L39) / [pkg-config](https://github.com/simbody/simbody/blob/master/cmake/pkgconfig/simbody.pc.in#L1) file. Both of these files are configured by CMake to provide the appropriate include dir for the platform you're on.

We expect that people building a project that uses Simbody to use CMake to set up the appropriate compiler/linker flags, etc. If a downstream project to Simbody chooses not to use CMake (or pkg-config) to set up the flags, then yes, the project will have to specify their own platform-dependent behavior. 

On UNIX, it is conventional to install all headers inside a `<project-name>` folder (e.g., `$prefix/include/simbody`), since the project is often installed in a system-wide directory. This is not the case on Windows (aside from conda), so an extra `simbody` directory would be weird.